python/tskit/trees.py

```diff
@@ -8592,6 +8592,132 @@ def genetic_relatedness_vector(
         )
         return out
 
+    def pca(
+            self,
+            n_components: int = 10,
+            iterated_power: int = 3,
+            n_oversamples: int = 10,
+            samples: np.ndarray = None,
+            individuals: np.ndarray = None,
+            centre: bool = True,
+            windows: list = None,
+            random_state: np.random.Generator = None,
```

Comment:
usually we just pass in a `seed`, any objections to doing that, instead?

Reply:
I changed the option from `random_state` to `random_seed` following msprime.